Gateway/Request/RefundDataBuilder.php

```diff
@@ -84,7 +84,7 @@ public function build(array $buildSubject)
         $pspReference = $payment->getCcTransId();
         $orderAmountCurrency = $this->chargedCurrency->getOrderAmountCurrency($payment->getOrder(), false);
         $currency = $orderAmountCurrency->getCurrencyCode();
-        $amount = $orderAmountCurrency->getAmount();
+        $amount = $buildSubjectAmount;
```

Comment:
is this formatted correctly to minor units?

Reply:
It gets formatted later, just like in branch 6 https://github.com/Adyen/adyen-magento2/blob/7.0.0-rc.2/Gateway/Request/RefundDataBuilder.php#L161